cirq/api/google/v2/metrics.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package cirq.api.google.v1;
+
+option java_package = "com.google.cirq.api.google.v2";
+option java_outer_classname = "CalibrationProto";
+option java_multiple_files = true;
+
+// A snapshot of the performance metrics for a quantum processor at a
+// particular time.
+message MetricSnapshot {
+
+  // The time the metrics were collected, in unix time (milliseconds since
+  // Epoch minus leap seconds). Metric collection take time, so this is the
+  // time at which all of the metrics have been collected.
+  uint64 collection_time_ms = 1;
```

Comment:
How about `timestamp_ms`? There's not really a single well defined time when all the metrics were collected, but there can be a well defined time when we stamped it.

Reply:
done